### ISSUE TITLE
Anonymous queries no longer available in bodhi-client 6.0

### ISSUE THREAD

**Issue by kparal** [fedora-infra/bodhi#4535]
```
$ pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ bodhi-client==6.0.0
$ ipython
In [1]: from bodhi.client.bindings import BodhiClient
In [2]: b = BodhiClient(staging=True)
In [3]: b.get_releases()
Authenticating... Please open your browser to:
...
```

I haven't found a way how to do anonymous queries anymore. That was the default behavior in `bodhi-client < 6` and now it seems to be gone? Maintaining auth tokens will be a headache for many tools depending on Bodhi (like [BlockerBugs](https://qa.fedoraproject.org/blockerbugs/)). Or am I missing something?

Thanks.

**Comment by kparal** [comment]
Thank you! Can you please update the build in test.pypi.org? I'll be happy to test this.

**Comment by abompard** [comment]
Thanks for the report! I can't easily upload to test.pypi.org but you can find RPMs here: https://github.com/fedora-infra/bodhi/actions/runs/2462762953

**Comment by kparal** [comment]
I'd need something installable with pip. Is there a wheel available as well? Or can you point me to a guide how to install or build this project easily (I see there is no `setup.py`)? Thanks!

**Comment by abompard** [comment]
Actually I just released 6.0.0 on PyPI, so you can pip install bodhi-client.

**Comment by kparal** [comment]
The patch seems to be working fine, thanks.
